### ISSUE TITLE
Wrong GUI scaling

### ISSUE THREAD

**Issue by mriza** [biolab/orange3#4763]
**Describe the bug**

Orange3 (version 3.25.0) was installed using conda 4.8.3 under Fedora 32. The display resolution is 1920x1080 with 100% scaling. Orange was displayed in 200% scaling when run using `python -m Orage.canvas` or using Anaconda Navigator.

**To Reproduce**

Steps to reproduce the behavior go **here**:
1. Install anaconda
2. activate base environment 'conda activate base'
3. python -m Orange.canvas    -OR-
4. Click Orange Launch button in Anaconda Navigator

**Orange version:**
Orange - 3.25.0

**Expected behavior**
Orange run in 100% scaling
 
**Screenshots**
![Screenshot from 2020-05-13 14-15-25](https://user-images.githubusercontent.com/1440628/81782636-54a9d900-9524-11ea-9d6c-78d5a0f5e1c4.png)

**Operating system:**
Linux Fedora 32

**Comment by mriza** [comment]
Temporary workaround for this bug is to set `QT_SCALE_FACTOR` and `QT_FONT_DPI` when invoking `python -m Orange.canvas` command.

I succesfully run Orange in acceptable resolution with
```
QT_SCALE_FACTOR = 50%
QT_FONT_DPI = 192 DPI
```

run this command in terminal :
QT_SCALE_FACTOR=0.5 QT_FONT_DPI=192 python -m Orange.canvas
![Screenshot from 2020-05-13 14-24-48](https://user-images.githubusercontent.com/1440628/81783445-85d6d900-9525-11ea-8ced-14b3239c9f17.png)

**Comment by ajdapretnar** [comment]
Similar to #2832.

**Comment by mriza** [comment]
> Similar to #2832.

Yes, similar but different, mine is not even HiDPI nor 4K

**Comment by ajdapretnar** [comment]
I don't think it is a temporary workaround, but a permanent solution. Orange is based on Qt and from what I know, that is their issue, not ours.

**Comment by mriza** [comment]
You're right. I will just create a launcher script for now, until this issue resolved by Qt.

**Comment by ajdapretnar** [comment]
I know it sucks, I was struggling with the same on Linux Mint and this way the only things that solved it. Sorry about that.

**Comment by mriza** [comment]
That's okay, I'll be patient

> You're right. I will just create a launcher script for now, until this issue resolved by Qt.
**Orange Application Launcher with Qt resolution workaround**

create **/opt/orange/orange**  :
```
#!/bin/bash
source ~/anaconda3/etc/profile.d/conda.sh 
conda activate base
QT_SCALE_FACTOR=0.5 QT_FONT_DPI=192 python -m Orange.canvas
```

create **/usr/share/application/Orange.desktop** :
```
[Desktop Entry]
Name=Orange 3
Exec=/opt/orange/orange
Type=Application
Icon=~/anaconda3/lib/python3.7/site-packages/Orange/canvas/icons/orange-canvas.svg
StartupNotify=true
Comment=Orange - Data Mining Fruitful and Fun.
Categories=Education;Science;
```

**Comment by mriza** [comment]
of course we can just disable DPI scaling instead of setting the scale manually by running
> QT_AUTO_SCREEN_SCALE_FACTOR=0 python -m Orange.canvas

**Comment by markotoplak** [comment]
@mriza, just to get a better understanding of this particular problem... You told us the resolution, but what is your screen size?

**Comment by mriza** [comment]
@markotoplak, its 1920x1080, I already mention this in the first comment. FYI Anaconda Navigator has setting item to disable HiDPI support, and I think Orange should also provide this option in the settings as well CMIIW.

**Comment by markotoplak** [comment]
@mriza, I was wondering what the physical monitor size (its diagonal) is, not the resolution.

**Comment by mriza** [comment]
I'm sorry @markotoplak :-) , It's a 15 inch laptop

**Comment by janezd** [comment]
Said on Friday:

- apparently we will need to add a checkbox (or something) to set scaling?
- @robertcv can reliably reproduce this, so he will work on this when he has time.

**Comment by robertcv** [comment]
Hey @mriza! I had the same problem (Ubuntu, 15 inch, 1080p). It despaired when upgrading to pyqt>=5.12. Try running `conda update pyqt` in your orange conda environment.

**Comment by janezd** [comment]
@robertcv will add dependency on pyqt>=5.12 and webengine.

**Comment by robertcv** [comment]
Fixed in #4849 .
